statsrunner/aggregate.py

```diff
@@ -18,9 +19,12 @@ def decimal_default(obj):
     else:
         return common.decimal_default(obj)
 
+
 def dict_sum_inplace(d1, d2):
-    if d1 is None: return
-    for k,v in d2.items():
+    """Merge values from dictionary2 into dictionary1."""
```

Comment:
`dictionary1` and `dictionary2` are not variable names in the function. As such, would suggest either:

* Switching to the `Args:` format to describe the arguments
* Changing the docstring to something along the lines of `Merge values from dictionaries d1 and d2`